### ISSUE TITLE
Query Breakdown of RAM Usage for Account

### ISSUE THREAD

**Issue by jafri** [EOSIO/eos#5134]
You can get RAM used on an account (standard) and RAM usage from chain through PR #3038 (https://github.com/EOSIO/eos/pull/3038).

However, you cannot query how it’s used. That would be a useful feature for account optimizations and deeper insight into where account RAM is being used.

**Comment by jjssoftware** [comment]
I agree, it would be very nice for both account owners and developers to be able to see how RAM within accounts is being used. This area is linked to issue #4537 

If we are eventually able to see in detail how RAM within accounts is being used, improved management of that RAM would also be very welcome. It's probably best that we save that idea for another issue and concentrate on this one for now.
